ptf/sainexthop.py

```diff
@@ -677,22 +721,19 @@ def setUp(self):
             vni_id_value=self.vni)
         encap_mappers_list = [self.encap_tunnel_map]
         decap_mappers_list = [self.decap_tunnel_map]
-        encap_mappers_objlist = sai_thrift_object_list_t(
+        self.encap_mappers_objlist = sai_thrift_object_list_t(
             count=1, idlist=encap_mappers_list)
-        decap_mappers_objlist = sai_thrift_object_list_t(
+        self.decap_mappers_objlist = sai_thrift_object_list_t(
             count=1, idlist=decap_mappers_list)
         self.tunnel = sai_thrift_create_tunnel(
             self.client, type=self.tunnel_type,
             encap_src_ip=sai_ipaddress(self.my_lb_ip),
-            decap_mappers=decap_mappers_objlist,
-            encap_mappers=encap_mappers_objlist,
+            decap_mappers=self.decap_mappers_objlist,
+            encap_mappers=self.encap_mappers_objlist,
             encap_ttl_mode=self.ttl_mode,
             decap_ttl_mode=self.ttl_mode,
             underlay_interface=self.urif_lb)
 
-    def runTest(self):
-        self.tunnelVniTest()
-
     def tearDown(self):
         sai_thrift_remove_tunnel(self.client, self.tunnel)
         sai_thrift_remove_tunnel_map_entry(self.client,
```

Comment:
self.decap_tunnel_map_entry is not an integer

Reply:
fixed in https://github.com/ravi861/SAI/pull/3 and https://github.com/ravi861/SAI/pull/2